OpenEdXMobile/src/main/java/org/edx/mobile/util/Config.java

```diff
@@ -395,7 +395,8 @@ public String getOAuthClientId() {
     }
 
     public String getOrganizationCode() {
-        return getString(ORGANIZATION_CODE, "");
+        String orgCode = getString(ORGANIZATION_CODE, "");
+        return orgCode.length() > 0 ? orgCode : null;
```

Comment:
@1zaman That wouldn't work if ORGANIZATION_CODE is set to an empty string. This way empty or non-existent will work correctly.